cmd/helm-operator/main.go

```diff
@@ -26,7 +26,8 @@ import (
 
 func main() {
 	root := cobra.Command{
-		Use: "helm-operator",
+		Short: "**NOTE** This executable is for use by operator-sdk and is not intended to be used manually",
```

Comment:
The binary can be run directly/manually, but only from within a Helm project.
```suggestion
		Short: "Reconcile an Helm operator project using helm",
		Long:  `This binary runs a Helm operator that reconciles Kubernetes resources
managed by the helm program. It and can be run either directly or from a Helm operator
project's image entrypoint
`,
```